### ISSUE TITLE
[markdown] Markdown lists aren't being themed correctly

### ISSUE THREAD

**Issue by martellaj** [Microsoft/vscode#104]
Making a custom theme using every possible scope for a Markdown list and can't style them. However, if I style "punctuation", the lists get styled.

**Comment by aeschli** [comment]
The markdown tokenizer only emits a very limited set of tokens form now.

**Comment by Tyriar** [comment]
@kieferrm I believe you came across this recently, can this issue be closed?

**Comment by kieferrm** [comment]
Yes, once the extension is in place, we'll close this.

**Comment by bpasero** [comment]
Done?

**Comment by Tyriar** [comment]
I just created a duplicate of this at https://github.com/Microsoft/vscode/issues/7789, it's more detail and more relevant though so I'll close this off in favor of it.
